styles/ContactUs.module.css

```diff
@@ -0,0 +1,70 @@
+.title {
+  text-align: center;
+  color: var(--primary-blue);
+  padding: var(--padding-lg);
+}
+.subtitle {
+  color: var(--primary-blue);
+}
+.subtitleBasedIn {
+  color: var(--primary-blue);
+  padding-bottom: 17.93px;
+}
+.mail {
+  text-decoration: none;
+  color: var(--primary-blue);
+  font-size: 1.25rem;
```

Comment:
Please use font size vars

Reply:
Don't  have the same size there I can use body3 because on the small screen is b3

Reply:
If it's a new size, you could create a new one or ask the UX team what's the right one

Reply:
done